docs/source/tutorial_tipsntricks.md

```diff
@@ -123,3 +123,40 @@ git repository.
 
 For more detailed instructions and features, see the
 [README](http://github.com/facebookresearch/ParlAI/blob/master/example_parlai_internal)
+
+Contributing
+------------
+
+ParlAI is maintained by a small team at Facebook, so we rely heavily on community contributions. We welcome pull requests with open wings!
+
+### Overview
+- Make your changes
+- Open a pull request
+- Iterate with reviewers, fix CI, add tests
+- Celebrate!
+
+#### Creating a Pull Request
```

Comment:
Based on the left side TOC, I suggest making all of them ###

Reply:
I fixed them to be consistent. I see that #### is a little over-indented. But if I make them ### they are almost the same size as the section header "Contributing" :/

Let me know if you feel strongly and I'll switch them to ###